commit 202a7792fbdf8763e8c50805b2bc15f007484f04
Author: popcornmix <popcornmix@gmail.com>
Date:   Sat Sep 8 14:49:16 2012 +0100

    Don't believe KDIR is required when building as part of kernel

Index: linux-3.2.46/drivers/usb/host/dwc_common_port/Makefile
===================================================================
--- linux-3.2.46.orig/drivers/usb/host/dwc_common_port/Makefile	2013-07-26 19:34:39.000000000 +0000
+++ linux-3.2.46/drivers/usb/host/dwc_common_port/Makefile	2013-07-26 19:36:25.000000000 +0000
@@ -30,9 +30,9 @@
 
 else
 
-ifeq ($(KDIR),)
-$(error Must give "KDIR=/path/to/kernel/source" on command line or in environment)
-endif
+#ifeq ($(KDIR),)
+#$(error Must give "KDIR=/path/to/kernel/source" on command line or in environment)
+#endif
 
 ifeq ($(ARCH),)
 $(error Must give "ARCH=<arch>" on command line or in environment. Also, if \
